dist/infragram.css

```diff
@@ -77,11 +77,6 @@ div#colorbar-container canvas#colorbar {
     height:auto;
   }
 }
-
-.container {
-  padding-left:unset;
-}
-
```

Comment:
These changes are not for this PR. Can you remove them? Thanks!

Reply:
Yeah! I will do that. Thank you.